packages/caravan-clients/src/client.ts

```diff
@@ -454,6 +501,32 @@ export class BlockchainClient extends ClientBase {
     }
   }
 
+  public async getTransaction(txid: string): Promise<TransactionDetails> {
+    try {
+      let txData: RawTransactionData;
+
+      if (this.type === ClientType.PRIVATE) {
+        const response = await bitcoindRawTxData(
+          this.bitcoindParams.url,
+          this.bitcoindParams.auth,
+          txid,
+        );
+        txData = response.result;
```

Comment:
this also needs to just be 
```suggestion
        txData = response;
```

Reply:
thanks for the fix :) 
so I change the bitcoin file to return the whole `response object` and not the `response.result object` as it made more sense and let the end consumer methods decide what they want to do with the whole object.

[here](https://github.com/caravan-bitcoin/caravan/pull/134/commits/8f5439e780508036e365cbbefb4b500ecd528283#diff-becbd4bbfbd061556d05bcab33227d25b2b48e6f34ce5cb03697b239772f3896R346)

Reply:
is anything except the result used though? I did a quick search around for `bitcoindRawTxData` and it looks like there are still places that are not accessing the result but before were expecting response.result. and I note in your [test](https://github.com/caravan-bitcoin/caravan/pull/134/files#diff-416e19bc7d09f1657b40c0ee25212cdb78cb020fa7f7038d639877e98bb38b9aR1259-R1261) you're only mocking out the `result` so it seems like that is actually all we care about so it might be worth switching it back.